### PR TITLE
Feature/add version

### PR DESCRIPTION
**Pull request by enys**
Add version files for nodePipeline

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
 
-# Development
+## Development
 - run tests: `npm run test` (uses Mocha)
 <!-- - generate coverage report: `npm run coverage` (uses istanbul) -->
 <!-- - generate API documentation `npm run doc` (uses JSDoc) -->
 - run code style checker: `npm run lint` (uses TSLint)
 
+## Versioning and publishing
+Versioning is handled by bump2version.
```
Comment (davidrapin) [inline README.md:11]: this deserves some more in-depth documentation (maybe in a wiki)

Comment (enys) [reply README.md:11]: yup will build a full documentation for nexus3 versioning + npm and docker repositories

Comment (enys) [reply README.md:11]: https://linkurious.atlassian.net/browse/OPS-72

**Comment by enys** [comment]
Ifit's good for you, we can merge.

**Pull request merged**